HW_1/Task_2.py

```diff
@@ -0,0 +1,10 @@
+# Задача 4: Петя, Катя и Сережа делают из бумаги журавликов. Вместе
+# они сделали S журавликов. Сколько журавликов сделал каждый
+# ребенок, если известно, что Петя и Сережа сделали одинаковое
+# количество журавликов, а Катя сделала в два раза больше журавликов,
+# чем Петя и Сережа вместе?
+
+s = int(input("Введите общее количество журавликов: "))
+a = (s / 6) * 4
```

Comment:
Приоритет у этих операций одинаков. К тому же тут нужно использовать целочисленное деление, тогда не придется использовать функцию `int`. И не плохо бы проверять переменную `s` на кратность 6
```suggestion
a = s // 6 * 4
```